python/src/trezorlib/messages.py

```
@@ -9807,6 +9797,49 @@ def __init__(
+        self.message = message
+        self.message_version = message_version
+        self.message_format = message_format
+        self.application_domain = application_domain
+
+
+class SolanaSignedMessage(protobuf.MessageType):
+    MESSAGE_WIRE_TYPE = 10105
+    FIELDS = {
+        1: protobuf.Field("signature", "bytes", repeated=False, required=True),
+        2: protobuf.Field("public_key", "bytes", repeated=False, required=True),
+    }
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)